README.md

```diff
@@ -13,6 +13,20 @@ Home of the [osu!](https://osu.ppy.sh/home) wiki.
 
 Please see [the "contributing" file](CONTRIBUTING.md) if you are interested in helping out with the project!
 
+### Maintainers
+
+Maintainers are users who contribute regularly and establish a solid role. These users are also referred to as wikifriends on the [osu!dev Discord](https://discord.gg/ppy) (`#osu-wiki` channel).
+
+| Name | Tag | Role |
+| :-- | :-- | :-- |
+| [cl8n](https://github.com/cl8n) | `clayton#1349` | Reviewer, code expert |
```

Comment:
There's also a couple of people who share this role with him as well, mind you.

Reply:
I only added people who are consistently active in discord/github.

If you’re talking about `code expert`, that probably needs to be reworded as it is vague atm.